apps/src/templates/sectionProgressV2/floatingScrollbar/floating-scrollbar.module.scss

```
@@ -24,14 +35,18 @@
 .scrollingContent {
   width: 100%;
   height: 100%;
-  // Add an invisible scroll bar to the content in order to enable two-finger scrolling.
   overflow-x: scroll;
-  scrollbar-width: none; /* Firefox */
-  -ms-overflow-style: none;  /* Internet Explorer 10+ */
```

Is this browser-specific stuff no longer needed? Did you test on all of them?

---

I switched the approach. Instead of hiding the default scrollbar all the time and using the custom scrollbar. We use the default scrollbar and hide the custom scrollbar when the bottom of the table is shown. When you can't see the bottom of the table, then we show the custom scrollbar. I suspect that this will be much better by default and fix some errors we are seeing with horizontal scrollbars not showing up.